src/app/asset-page/artstor-viewer/artstor-viewer.component.scss

```diff
@@ -69,6 +69,11 @@ $desktop: 992px;*/
             background-color: $color-gray-3 !important;
         }
 
+        &:focus {
+            outline: auto 2px Highlight !important;
```

Comment:
I don't think I have ever seen the `Highlight` before. Is that custom for AIW?

Reply:
No, it's what I was trying to simulate the "blue border" in firefox 😂

Reply:
Does this end up working in firefox or do we still end up with the gray dotted border?

Reply:
Yes, this ends up working in firefox.